SU2_CFD/src/numerics/CTrapezoidalMap.cpp

```diff
@@ -0,0 +1,269 @@
+/*!
+ * \file TrapezoidalMap.cpp
+ * \brief Implementation of the trapezoidal map for tabulation and lookup of fluid properties
+ * \author D. Mayer, T. Economon
+ * \version 7.3.1 "Blackbird"
+ *
+ * SU2 Project Website: https://su2code.github.io
+ *
+ * The SU2 Project is maintained by the SU2 Foundation
+ * (http://su2foundation.org)
+ *
+ * Copyright 2012-2022, SU2 Contributors (cf. AUTHORS.md)
+ *
+ * SU2 is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * SU2 is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with SU2. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "../../Common/include/option_structure.hpp"
+
+#include "../include/numerics/CTrapezoidalMap.hpp"
+
+using namespace std;
+
+CTrapezoidalMap::CTrapezoidalMap(vector<su2double> const& samples_x, vector<su2double> const& samples_y,
+                                 vector<vector<unsigned long> > const& edges,
```

Comment:
clang format

Reply:
clang-format says this is OK.  What would you like this to be? Every argument on a line when the list of arguments is too long? Would you like to see the behavior of:
BinPackParameters:false
If false, a function declaration’s or function definition’s parameters will either all be on the same line or will have one line each.

Reply:
Having the const first, but if clang format doesn't complain it's fine I guess, even if not super consistent.

Reply:
I noticed that BinPackParameters:false does this (put const in front) as well, maybe we should add it to .clang-format?